add-on/src/lib/log.js

```diff
@@ -0,0 +1,14 @@
+const root = 'ipfs-companion'
+
+const debug = require('debug')
+
+module.exports = (loggerName) => {
+  const log = debug(`${root}:${loggerName}`)
+  log.debug = debug(`${log.namespace}:debug`)
```

Comment:
IMHO `log` and `log.error` are enough.

Reply:
Ok, if we just drop `log.debug` and use `log` everywhere,
should we set the default to:
-  hide non-errors by via something like  `localStorage.debug = '*:error'` 
- or print all Companion logs `localStorage.debug = 'ipfs-companion:*'` 

?